doc/man3/EVP_PKEY_todata.pod

```diff
@@ -0,0 +1,64 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_todata, EVP_PKEY_export
+- functions to return keys as an array of key parameters
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ int EVP_PKEY_todata(const EVP_PKEY *pkey, int selection, OSSL_PARAM **params);
+ int EVP_PKEY_export(const EVP_PKEY *pkey, int selection,
+                     OSSL_CALLBACK *export_cb, void *export_cbarg);
+
+=head1 DESCRIPTION
+
+The functions described here are used to extract B<EVP_PKEY> key values as an
+array of B<OSSL_PARAM>.
+
+EVP_PKEY_todata() extracts values from a key I<pkey> using the I<selection>.
+I<selection> is described in L<EVP_PKEY_fromdata(3)/Selections>.
+L<OSSL_PARAM_free(3)> should be used to free the returned parameters in
+I<*params>.
+
+EVP_PKEY_export() is similiar to EVP_PKEY_todata() but uses a callback
+I<export_cb> that gets passed the value of I<export_cbarg>.
+See L<openssl-core.h(7)> for more information about the callback. Note that the
+B<OSSL_PARAM> array that is passed to the callback is not persistent after the
+callback returns. Use EVP_PKEY_todata() if persistent data is required.
```

Comment:
s/Use EVP_PKEY_todata() if persistent data is required/You must preserve the items of interest, or use EVP_PKEY_todata() if persistence is required/

Reply:
Maybe copy instead of preserve?

Reply:
You can't know exactly how they want to do that.  I can imagine have a callback that writes the values to a BIO that's passed as the cb arg.